client/src/components/Header.js

```diff
@@ -228,11 +241,60 @@ export default class Header extends Component {
     }
 
     renderLogo() {
-        let imageLink = this.props.loggedIn ? this.props.schoolLogo : require('./logo.png')
+        let imageLink = this.props.loggedIn && this.state.currRole !== 'ADMIN' ? 
+            this.props.schoolLogo : require('./logo.png');
         return (
                 <Image centered src={imageLink} size='small'/>                
         )
     }
+
+    renderRoleDropdown() {
+        return(
+            <div>
+            {this.state.availableRoles.length > 1 && 
+            <>
+                <Label basic >Current Role:
+                    <Dropdown
+                        style={{'marginLeft': '5px'}}
+                        selection
+                        options={this.state.availableRoles}
+                        value={this.state.currRole}
+                        onChange={this.changeRole.bind(this)}
+                    />
+                </Label>
+                
+            </>
+            }
+            </div>
+        )
+    }
+
+    changeRole = (e, { value }) => {
+        this.setState({ currRole: value })
```

Comment:
We might want to do `this.setState({ currRole: value} ,() => this.props.liftRole(value))`